openslides/core/static/js/core/base.js

```diff
@@ -808,6 +808,7 @@ angular.module('OpenSlidesApp.core', [
                                 id: value.id,
                             };
                         } else if (
+                            value.name != 'voting/prompt' &&
```

Comment:
This is a non-core feature. So I think we need a new interface for plugins to use this method properly. Please create a new issue for this and add a short comment here about the problem and refer to the new issue.